js/popover.js

```diff
@@ -65,10 +65,16 @@
     var $e = this.$element
     var o  = this.options
 
-    return $e.attr('data-content')
+    var content = $e.attr('data-content')
       || (typeof o.content == 'function' ?
             o.content.call($e[0]) :
             o.content)
+
+    if (['[object Function]', '[object Object]', '[object String]'].indexOf(Object.prototype.toString.call(content)) === -1) {
```

Comment:
5bf13914e49942409c0b21aeaf2ed756c41bd498 implies that we do support it, although there are no tests and it's missing from the docs as you mentioned.